### ISSUE TITLE
Add hourly timestamp to explore page date slider

### ISSUE THREAD

**Issue by jerikac** [NASA-IMPACT/veda-config-ghg#31]
EMIT Plume dataset will occasionally have multiple plume observations in a single day, requiring the need to distinguish observations by the hour. This is a request specific to the EMIT Methane Plume Emission datasets. This is a high-priority ask from the EMIT team. 
![image](https://github.com/NASA-IMPACT/veda-config-ghg/assets/98853455/dbd34ff3-85ef-43cb-8155-676913637fad)

**Comment by jerikac** [comment]
@aboydnw @hanbyul-here Could you help with assigning this issue?

**Comment by aboydnw** [comment]
This will probably take some design thinking, so I assigned to both @hanbyul-here and @ricardoduplos 
I also added this to the [spreadsheet](https://docs.google.com/spreadsheets/d/1u5eYDI_EmL3sSttYkTm91_lEeK2tFsgXH3D22wczmrA/edit#gid=88759098) and gave a priority of 2. Let me know if that prioritization is incorrect @jerikac @deborahUAH @Jeanne-le-Roux

**Comment by danielfdsilva** [comment]
The dashboard is not setup to handle sub day precision, as the different date selectors do not include a hour option. There are also several calculation being done using dates that use daily precision discarding any hour level "noise".

This feature has a very high level of effort as it requires reviewing several components to include time (while ensuring that other less precise datasets continue to function), building time pickers and designing a proper display for these detailed views.

There are also constrains on the data-services side given the added granularity of the data. @anayeaye perhaps you could weigh in a bit here, or link to some other source.

**Comment by Jeanne-le-Roux** [comment]
It's understandable that building in this feature is complex and should be well thought out. While I don't think we need to solve this for R1, we will have to figure out a way to deal with it for GHG Center. There are several features associated with interacting with this type of point source/event data on a map that the current explore view isn't well equipped to deal with. We will need to scope out (perhaps as an epic) for the full release all of the functionality needed for this type of data, since it will be important for the Center and similar data will continue to increase and be produced by multiple sensors.

**Comment by anayeaye** [comment]
@danielfdsilva I proposed this [process for browsing spatially and temporally sparse collections](https://github.com/NASA-IMPACT/veda-config-ghg/issues/50#issuecomment-1689020429) that might support browsing the EMIT data without adding hourly time selection. It is a little cumbersome but the process may inspire a better solution.

Thanks for flagging this for indexing concerns about granularity, too, I had to ask about a sensible cutoff for pgstac: when we expect a collection's item count to exceed 1 million, we will need to adjust how partitions are generated _before_ items are ingested. From the currently loaded data it looks like we will be safe.
